gemfiles/modular/style.gemfile

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+# We run rubocop on the latest version of Ruby,
+#   but in support of the oldest supported version of Ruby
+
+gem "rubocop-lts", "~> 0.1", ">= 0.1.1" # Style and Linting support for Ruby >= 1.8
```

Comment:
`rubocop-lts` is how I package all of the rules I've found to support every version of Ruby since v1.8, where every even version of the gem corresponds to a specific minor version of Ruby, (odd releases are deprecated, or non-extant).

See: https://rubocop-lts.gitlab.io/